docs/en/operations/settings/settings.md

```diff
@@ -3665,6 +3665,16 @@ Possible values:
 
 Default value: `0`.
 
+## s3_ignore_file_doesnt_exist {#s3_ignore_file_doesnt_exist}
```

Comment:
Let's also add settings `s3_validate_request_settings` and `azure_skip_empty_files` to docs